shell/sct_run_batch.sh

```diff
@@ -91,11 +92,12 @@ if [ "$#" -ne 2 -a "$#" -ne 3 ]; then
  fi
 
 # getopts processes flag input parmaeters
-par='false'
-while getopts "ph" opt "$3"; do
+par='true'
```

Comment:
in shell language, Bool variables exist so i recommend you use it instead of using string type.